R/EngineGraphQL.R

```diff
@@ -69,6 +69,48 @@ EngineGraphQL <- R6::R6Class("EngineGraphQL",
            file_path = file_path
          )
        return(files_table)
+     },
+
+     #' @description Method to get releases.
+     #' @param org An organization.
+     #' @param repos A vector of repositories.
+     #' @param date_from A starting date of releases.
+     #' @param date_until An end date for releases.
+     #' @param settings A list of  `GitStats` settings.
+     #' @param storage A storage of `GitStats` object.
+     #' @return A table of commits.
```

Comment:
When we have parameters with default values, can we describe what they stand for?